controllers/suite_test.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 package controllers
 
-//import (
```

Comment:
@georgettica wonder if it makes sense to remove this file completely if we do not use it. What do you think? Can we remove this savely? In the worst case, we still have it in our git history.

Reply:
yeah, this can happen no problem.
my idea was to make `operator-sdk` be able to regenerate and update it's generated data.

but it's a dream I am not stiving on making, so NP